Sources/Brave/Frontend/Browser/BrowserViewController/BrowserViewController+P3A.swift

```diff
@@ -93,6 +95,75 @@ extension BrowserViewController {
     
     usage.recordReturningUsageMetric()
   }
+  
+  func recordAccessibilityDisplayZoomEnabledP3A() {
+    // Accessibility Q1 New P3A iOS - Do you have iOS display zoom enabled?
+    let isDisplayZoomEnabled = UIScreen.main.scale < UIScreen.main.nativeScale
+    
+    UmaHistogramBoolean("Brave.Accessibility.DisplayZoomEnabled", isDisplayZoomEnabled)
+  }
+  
+  func recordAccessibilityDocumentsDirectorySizeP3A() {
+    // Accessibility Q4 New P3A iOS - Documents directory size
+    func fetchDocumentsAndDataSize() -> Int? {
+      let fileManager = FileManager.default
+      
+      guard let documentsDirectory = FileManager.default.urls(for: .documentDirectory, in: .userDomainMask).first else {
+        return nil
+      }
+
+      do {
+        if let documentsDirectorySize = try fileManager.directorySize(at: documentsDirectory){
```

Comment:
Do we only want to consider the `Documents` directory specifically? Or was this question trying to answer the total size of the sandbox which would include `tmp`

Reply:
This question here is answering `Documents` directory specifically which was clear in the document. 
Probably @iccub can clarify this for us.

Reply:
What do you think?
Sandbox would be better imo